### PR TITLE
Mapdev197 welsh text export

### PR DESCRIPTION
**Pull request by lgarvey**
Welsh text export - all text in all templates and python files has been wrapped in i18n tags and a django.po file has been generated for the cy / Welsh language.  

We need to merge these changes in quickly so that further dev tasks make use i18n, but at the same time given every string on the site has been changed we need to be careful about testing everything.

### CODE REVIEW

**Comment by froddd** [comment]
Checked all screens local vs staging and it all seems good to me. Passing this to Ian now!

File: apps/plea/forms.py [hunk 1]
```diff
@@ -32,39 +33,39 @@ class BasePleaStepForm(forms.Form):
...
 
 
 class YourDetailsForm(BasePleaStepForm):
     name = forms.CharField(widget=forms.TextInput(attrs={"class": "form-control"}),
                            max_length=100, required=True, label="Full name",
-                           help_text="On page 1 of the pack we sent you",
+                           help_text=_("On page 1 of the pack we sent you"),
                            error_messages={"required": ERROR_MESSAGES["FULL_NAME_REQUIRED"]})
     contact_number = forms.CharField(widget=forms.TextInput(attrs={"class": "form-control"}),
                                      max_length=30, required=True, label="Contact number",
```
Comment (elbeanio) [inline apps/plea/forms.py]: Does this need translating?

File: apps/plea/forms.py [hunk 2]
```diff
@@ -97,45 +98,45 @@ class CompanyDetailsForm(BasePleaStepForm):
...
+    SE_PERIOD_CHOICES = (("Weekly", _("Weekly")),
+                         ("Fortnightly", _("Fortnightly")),
+                         ("Monthly", _("Monthly")),
+                         ("Self-employed other", _("Other")),)
+    BEN_PERIOD_CHOICES = (("Weekly", _("Weekly")),
+                         ("Fortnightly", _("Fortnightly")),
+                         ("Monthly", _("Monthly")),
+                         ("Benefits other", _("Other")),)
     YES_NO = (("Yes", "Yes"),
               ("No", "No"))
```
Comment (elbeanio) [inline apps/plea/forms.py]: Yes / No has been done elsewhere so I assume it needs it here as well

File: manchester_traffic_offences/templates/plea/review.html [hunk 3]
```diff
@@ -28,33 +30,36 @@ <h2 class="heading-medium">You need to fix the errors on this page before contin
...
             </tr>
             <tr>
-                <th class="label">Number of charges against you</th>
+                <th class="label">{% trans "Number of charges against you" %}</th>
                 <td>{{ case.number_of_charges }}</td>
             </tr>
             <tr>
-                <th class="label">Plea on behalf of a company?</th>
+                <th class="label">{% trans "Plea on behalf of a company?" %}</th>
                 <td>{{ case.company_plea|yesno:"Yes,No" }}</td>
```
Comment (elbeanio) [inline manchester_traffic_offences/templates/plea/review.html]: Is there a way of translating this?

File: manchester_traffic_offences/templates/plea/review.html [hunk 4]
```diff
@@ -181,39 +188,39 @@ <h1 class="heading-xlarge">Confirm your plea{{ case.number_of_charges|pluralize
...
+                    <th class="label">{% trans "Your Job" %}</th>
                     <td>{{ your_money.your_job }}</td>
                 </tr>
                 <tr>
-                    <th class="label">You get paid</th>
+                    <th class="label">{% trans "You get paid" %}</th>
                     <td>
-                        {% if your_money.self_employed_pay_period == "Self-employed other" %}
-                        Other
+                        {% if your_money.self_employed_pay_period == "self-employed other" %}
```
Comment (elbeanio) [inline manchester_traffic_offences/templates/plea/review.html]: Can you change "self-employed other" back to "Self-employed other" ?

@froddd noticed it and wanted me to credit him :)

File: manchester_traffic_offences/templates/plea/review.html [hunk 5]
```diff
@@ -280,28 +287,28 @@ <h1 class="heading-xlarge">Confirm your plea{{ case.number_of_charges|pluralize
             <thead>
                 <tr>
                     <th class="header">Your expenses</th>
```
Comment (elbeanio) [inline manchester_traffic_offences/templates/plea/review.html]: Needs translating

**Pull request merged**